### ISSUE TITLE
Installation: Meshery bash script errors on permission check

### ISSUE THREAD

**Issue by leecalcote** [layer5io/meshery#644]
**Description**
System: macOS
Shell: zsh
Status: `mesheryctl` is already installed on this system before running script. `mesheryctl` was installed using brew on this system.

**Output of script:**
<img width="879" alt="Screen Shot 2020-01-31 at 12 10 26 PM" src="https://user-images.githubusercontent.com/7570704/73563492-0d8b5280-4423-11ea-8277-5dfa6adccf58.png">

Permissions of the existing `mesheryctl` binary:
```
➜  ~ ls -l /usr/local/bin/mesheryctl
-rwxr-xr-x  1 root  staff  10703744 Jan 17 11:02 /usr/local/bin/mesheryctl
```

Let's also address the other feedback included in the review of PR #642.

**Comment by leecalcote** [comment]
// @anandrkskd @nupur-thakur

**Comment by anandrkskd** [comment]
Got it!

**Comment by leecalcote** [comment]
PR #648 addresses part of this issue (#642). I'm reopening this issue (#642) in order to track fixes for the additional feedback in #642.

**Comment by leecalcote** [comment]
Fixed! Thanks, @nupur-thakur, @anandrkskd
